adapter_pipelines/10x/adapter_example_bundle_specific.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Wrapper10xCount.bundle_uuid": "c1677888-59fb-43bd-b3e3-05c274cdb11a",
```

Comment:
I really love the renaming from wrapper to adapter, which is more accurate. But should we also names inside of the code to adapt to this change? I know that would be a lot more refactoring though.

Reply:
Good idea, done